backend-rs/src/sql.rs

```diff
@@ -0,0 +1,61 @@
+use chrono::NaiveDateTime;
```

Comment:
@Centeno448 While I think the modularization of the API makes the code more readable, I am not completely convinced about this "sql" and "search" abstractions for a few reasons:

- The code on these modules is not reused across different places in the code
- It creates a level of indirection

Following the [Locality of Behavior](https://htmx.org/essays/locality-of-behaviour/) principle, I would recommend making things easier for the reader by only extracting things to a separate module if they appear in multiple places in the code.